src/corehost/cli/runtime_config.cpp

```diff
@@ -71,6 +72,21 @@ bool runtime_config_t::parse_opts(const json_value& opts)
         m_prerelease_roll_fwd = prerelease_roll_fwd->second.as_bool();
     }
 
+    auto roll_fwd_on_no_candidate_fx = opts_obj.find(_X("rollForwardOnNoCandidateFx"));
+    if (roll_fwd_on_no_candidate_fx != opts_obj.end())
+    {
+        m_roll_fwd_on_no_candidate_fx = roll_fwd_on_no_candidate_fx->second.as_bool();
```

Comment:
It will only work with boolean values. I defined it like this because that's how it was being done to patch and pre-release roll forwards properties. I tried to keep the pattern.